crates/qt-build-utils/src/lib.rs

```diff
@@ -306,8 +306,73 @@ impl QtBuild {
         .to_string()
     }
 
+    fn cargo_link_qt_library(
+        &self,
+        builder: &mut cc::Build,
+        name: &str,
+        prefix_path: &str,
+        lib_path: &str,
+        link_lib: &str,
+        prl_path: &str,
+    ) {
+        println!("cargo:rustc-link-lib={link_lib}");
+
+        match std::fs::read_to_string(&prl_path) {
```

Comment:
run `cargo clippy` to see suggestions here that CI is failing on

Reply:
Oops was sure I did that. Should be fixed.